modules/cluster-logging-collector-log-forward-cloudwatch.adoc

```diff
@@ -0,0 +1,294 @@
+            "ingestionTime": 1629422744016
+        },
+...
+----
+
+.Example: Replacing the default `infrastructureName` prefix in log group names
+
+You can replace the default `infrastructureName` prefix, `mycluster-7977k`, in the log group names by specifying a `groupPrefix` field in the `ClusterLogForwarding` CR:
+
+[source,yaml]
+----
+cloudwatch:
+    region: us-east-2
+    groupBy: logType
+    groupPrefix: demo-group-prefix
+----
+
+The value of `groupPrefix` replaces the default `infrastructureName` prefix:
+
+[source,terminal]
+----
+$ aws logs describe-log-groups | jq .logGroups[].logGroupName
+"demo-group-prefix.application"
+"demo-group-prefix.audit"
+"demo-group-prefix.infrastructure"
+----
+
+.Example: Creating log groups for application namespace names
+
+You can create separate log groups for each application namespace. The names of the resulting log groups are based on the names of the application namespaces. Use this approach when successive namespace objects with the same name are considered "equivalent".
+
+To create application log groups whose names are based on the names of the application namespaces, you set the value of the `groupBy` field to `namespaceName` in the `ClusterLogForwarder` CR:
+
+[source,terminal]
+----
+cloudwatch:
+    region: us-east-2
+    groupBy: namespaceName
+----
+
+[NOTE]
+====
+Destroying a namespace object and creating a new one that has the same name produces a new log group. To distinguish these log groups from each other, create log groups that are named after UUIDs instead of namespace names. See the next example, "Example: Creating log groups for application namespace UUIDs."
+====
+
+The `groupBy` field affects the application log group only. It does not affect the `audit` and `infrastructure` log groups.
+
+In AWS, the namespace name appears at the end of each log group name. Because there is a single application namespace, "app", the following output shows a new `mycluster-7977k.app` log group instead of `mycluster-7977k.application`:
+
+[source,terminal]
+----
+$ aws logs describe-log-groups | jq .logGroups[].logGroupName
+"mycluster-7977k.app"
+"mycluster-7977k.audit"
+"mycluster-7977k.infrastructure"
+----
+
+If the cluster in this example had contained multiple application namespaces, the output would show multiple log groups, one for each namespace.
+
+The `groupBy` field affects the application log group only. It does not affect the `audit` and `infrastructure` log groups.
+
+.Example: Creating log groups for application namespace UUIDs
+
+Destroying a namespace object and creating a new one that has the same name produces a new log group. To distinguish these log groups from each other, you can create log groups that are named after UUIDs instead of namespace names.
```

Comment:
We're repeating that confusing note from above here.
Actually, that line is gone from https://github.com/openshift/enhancements/blob/master/enhancements/cluster-logging/forward_to_cloudwatch.md, too.

Reply:
Updating. Thanks.